src/char/char.c

```diff
@@ -1358,106 +1358,116 @@ int char_rename_char_sql(struct char_session_data *sd, uint32 char_id)
 	return 0;
 }
 
-int char_check_char_name(char * name, char * esc_name)
+/**
+* Function that check if a given name is valide within config
+* currently more oriented to for newchar creation
+* @param inout name : Char name to check
+return e_makechar_error
+*/
```

Comment:
Function that checks if the given name is within the valid parameters.
'@param' name : Character name to check
'@return' e_makechar_error

-- Obviously don't include the quote, I just had to put them so we don't tag users. :)